SemanticMediaWiki.php

```diff
@@ -39,6 +39,8 @@ public static function initExtension( $credits = [] ) {
 			define( 'SMW_EXTENSION_LOADED', true );
 		}
 
+		include_once __DIR__ . '/src/Aliases.php';
```

Comment:
What does this have to do with 7.3 compat?

Reply:
Nothing. Thanks. This is what happens when you have two different clones of SMW open and modifying them at the same time.